tests/unit/Loading.test.js

```diff
@@ -14,7 +14,12 @@ test('Loading renders', () => {
         </Loading>
     );
 
-    expect(loading.html()).toMatchSnapshot('Loading with is_loading=true');
+    expect(
+        loading
+            .find('.dash-spinner')
+            .parent()
+            .html()
+    ).toMatchSnapshot('Loading with is_loading=true');
```

Comment:
I don't see the corresponding snapshot baseline in this PR

Reply:
This snapshot is already here: https://github.com/plotly/dash-core-components/blob/803b67b5f234aa70f8e536f7c93bd730f8680978/tests/unit/__snapshots__/Loading.test.js.snap#L224

The baseline is the same, but the test itself had to change because I moved things "down" in the DOM.